src/document-converter.ts

```diff
@@ -1085,6 +1120,15 @@ export class DocumentConverter {
     }
     const usedIdentifiers = collectIdentifierNames(program, ignoredIdentifiers);
 
+    const allImportReferences = new Set();
+    for (const refs of importedReferences.values()) {
+      for (const ref of refs) {
```

Comment:
code ⛳️ : `allImportReferences.add(...refs)` lets you skip the nested for-loop

Reply:
I don't think `add` accepts multiple arguments, does it?

Reply:
bah, you're totally right. nvm!